.github/golangci.yml

```diff
@@ -56,24 +58,27 @@ linters-settings:
     checks: [ "all", "-ST1022", "-ST1003" ]
   errorlint:
     asserts: false
+
   gocritic:
     enabled-tags:
       - diagnostic
       - experimental
       - opinionated
       - performance
       - style
+
   forbidigo:
     forbid:
       - p: '^regexp\.(Match|MatchString)$'
-        msg: it will re-compile the regexp for each execution; compile the regexp with regexp.Compile and store it as a singleton
```

Comment:
iirc the error message came out better written out (in context) as I wrote it

Reply:
```
cmd/gno/main.go:26:2: use of `regexp.MatchString` forbidden because "it will re-compile the regexp for each execution; compile the regexp with regexp.Compile and store it as a singleton" (forbidigo)
```

Reply:
Good call 💯 


579bd75